task/buildah/0.1/buildah.yaml

```diff
@@ -9,10 +9,14 @@ metadata:
   name: buildah
 spec:
   description: |-
-    Buildah task builds source into a container image and then pushes it to a container registry.
-    Buildah Task builds source into a container image using Project Atomic's Buildah build tool.
-    It uses Buildah's support for building from Dockerfiles, using its buildah bud command.
-    This command executes the directives in the Dockerfile to assemble a container image, then pushes that image to a container registry.
+    Buildah task:
+      - builds source into a container image
+      - generates a SBOM file
+      - injects the SBOM file into container image
+      - pushes the container image to container registry
+      - pushes SBOM as container image to container registry
+      - (optionally) for Java builds when JBSConfig is available it triggers rebuilds of java artifacts
+      - (optionally) uses artifacts from prefetch-dependencies to build in hermetic environment
```

Comment:
Out of curiosity: would this list be formatted correctly on the UI?
![image](https://user-images.githubusercontent.com/43273420/228281508-302829d3-df86-4ee0-824a-2cb23d5d6d06.png)